gravitee-apim-console-webui/src/index.ts

```diff
@@ -72,14 +72,14 @@ function fetchData() {
     .then((responses: any) => {
       const consoleResponse = responses[0];
       const uiCustomization = responses[1];
-      if (uiCustomization && uiCustomization.data) {
-        customizeUI(uiCustomization.data);
-      }
 
       const constants = _.assign(ConstantsJSON);
       constants.org.settings = consoleResponse.data;
-      if (uiCustomization?.data?.title) {
+
+      if (uiCustomization?.data) {
+        customizeUI(uiCustomization.data);
         constants.org.settings.management.title = uiCustomization.data.title;
+        constants.customization = uiCustomization.data;
```

Comment:
If we add it to constants, then will it limit if it's dynamic? I don't think that it's a big deal to have it in Constants. But if we add it there, then we can undo making the Menu Observables and just check the Constants.

Reply:
You're right. I have simplified the code to use Constants